### PR TITLE
Freely Create Attestations (relies on custom trust anchor)

### PR DESCRIPTION
**Pull request by JesusMcCloud**

### CODE REVIEW

**Comment by JesusMcCloud** [comment]
just check if i didn't mess anything up, it's your contribution anyway.

**Comment by nodh** [comment]
I'm confused, there are 14 commits, but only some minor changes in 4 files ... and the title of this MR doesn't align with the changes.

File: android-attestation/build.gradle.kts [hunk 1]
```diff
@@ -133,9 +133,10 @@ publishing {
     }
     repositories {
         mavenLocal()
+        //can be useful for locally verifying publication works as expected
         maven {
             url = uri(layout.projectDirectory.dir("..").dir("repo"))
-            name = "local"
+            name = "staging"
```
Comment (nodh) [inline android-attestation/build.gradle.kts:139]: Isn't it `local` everywhere else?

**Comment by JesusMcCloud** [comment]
> I'm confused, there are 14 commits, but only some minor changes in 4 files ... and the title of this MR doesn't align with the changes.
Apparently, adverse effects of sleep depravation are  a thing. Sorry!

**Pull request closed without merging**